solana-programs/audius_eth_registry/src/processor.rs

```diff
@@ -13,8 +13,12 @@ use solana_program::{
     msg,
     pubkey::Pubkey,
     sysvar,
+    sysvar::clock::Clock,
+    sysvar::Sysvar,
 };
 
+const MAX_TIME_DIFF: i64 = 2000000; // this is about 3 minutes
```

Comment:
I recommend using UnixTimestamp instead of i64 (they are the same thing internally), simply because a signed integer is frowned upon in smart contracts.